src/mock-data.js

```diff
@@ -1,27 +1,19 @@
 // dev
 import uuid from "uuid";
+import Dates from "./utilities/dates";
+
+const createNewNote = ({ title, dateCreated = new Date(), body }) => ({ id: uuid(), title, dateCreated: Dates.format(dateCreated), body });
```

Comment:
I think it is best to not format the date in the data source, rather on display. This lets us keep a friendly timestamp for the backend.